app-typescript/components/DateTimePicker.tsx

```diff
@@ -70,7 +70,8 @@ export class DateTimePicker extends React.PureComponent<IProps> {
                             this.handleDateChange(val?.toString());
                         }}
                         dateFormat={this.props.dateFormat}
-                        label={this.props.label}
+                        inlineLabel
```

Comment:
modify the interface to make label optional and pass these props if it is not set. If a label is passed from props it should be rendered. Now you hide it unconditionally

Reply:
Please don't forget accessibility. We should be able to define a label while choosing to hide it visually, making it accessible for screen readers. I'm not sure if this is the case here, but I'm mentioning it as a general reminder.